operator/v1alpha1/register.go

```diff
@@ -32,6 +32,7 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	metav1.AddToGroupVersion(scheme, GroupVersion)
 
 	scheme.AddKnownTypes(GroupVersion,
+		&ClusterCSIDriver{},
```

Comment:
Do we need to add ClusterCSIDriverList? Is that object required?

Reply:
added.